app/views/users/index.html.erb

```diff
@@ -1 +1,17 @@
-<h1>List of users</h1>
+<main>
+  <section>
+    <ul>
+      <% @users.each do |user| %>
+        <%= link_to user_user_posts_path(user) do %>
```

Comment:
- [ ] Ths link should return the users#show page, but it is returning the posts#index page. Please 🙏🏽 correct the path to open the correct page.